lib/commands/team.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable standard/no-callback-literal */
+
+const figgyPudding = require('figgy-pudding')
+const libteam = require('libnpmteam')
+const log = require('npmlog')
+const npmConfig = require('../config.js')
+
+const Team = module.exports = {
+  command: 'team',
+  describe: 'Used to manage teams in organizations, and change team memberships. Does not handle permissions for packages.',
```

Comment:
You can remove the last sentence here.

Reply:
Sentence removed regarding handling permissions.